test/app/collections/Products.spec.coffee

```diff
@@ -0,0 +1,14 @@
+describe 'ProductModel', ->
+  # Temporarily turn ajax back on for a real call to /db/products
+  beforeEach -> jasmine.Ajax.uninstall()
+  afterEach -> jasmine.Ajax.install()
+  it 'basic_subscription products have payPalBillingPlanID set', (done) ->
+    $.ajax("/db/products")
+    .done (data, textStatus, jqXHR) =>
+      for product in data
+        continue unless /basic_subscription/.test(product.name)
+        expect(product.payPalBillingPlanID).toBeDefined()
+      done()
+    .fail (jqXHR, textStatus, errorThrown) =>
+      console.error(jqXHR, textStatus, errorThrown)
+      done(textStatus)
```

Comment:
basic_subscription products always need `payPalBillingPlanID` set.  Is there a way to do that via schema validation?